templates/python/.github/workflows/python-poetry-release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Default reusable workflow
```

Comment:
Please give it a more meaningful name, perhaps related to the Poetry release workflow you're calling. I would propose "Publish" or something similar

Reply:
"Publish" sounds good, changed it to that.